src/components/Camera/style.css

```diff
@@ -176,3 +219,46 @@
   text-decoration: underline;
   cursor: pointer;
 }
+
+.absoluteBottom() {
+  position: absolute;
+  bottom: 48px;
+  left: 16px;
+  right: 16px;
+}
+
+.captureActions {
+  @media (--small-viewport) {
+    .absoluteBottom();
+  }
+
+  .livenessCamera & {
+    .absoluteBottom();
+  }
+}
+
+.whiteText() {
+  color: #fff;
+  font-size: 14px;
+  margin-bottom: 10px;
+}
+
+.captureActionsHint {
+  @media (--small-viewport) {
```

Comment:
why two selectors `@media (--small-viewport)` and `.livenessCamera & ` if the content is the same?

Reply:
Same reason discussed [here](https://github.com/onfido/onfido-sdk-ui/pull/365#discussion_r203092465).